public_html/templates/base.html

```diff
@@ -75,25 +77,39 @@
 
 	{% block main %}
 		<section class="filters container-fluid">
-			<form>
-				<fieldset class="form-group filters-fieldset text-center">
-					{% for type, description in filterTypes %}
-						<span class="radio-inline">
+			<form id="filters-form">
+				<fieldset class="form-group filters-fieldset col-lg-6">
```

Comment:
Here I don't think `lg` vs `sm` makes a difference as there are no columns, just two rows, which begs the question if we even need gird classes. I agree the left-alignment looks weird but I tried centering and it didn't look too great either. Here's a screenshot:
![screen shot 2016-07-21 at 12 50 13 am](https://cloud.githubusercontent.com/assets/366374/17011549/23d8211a-4edd-11e6-83d9-a874cfc8c1d1.png)
If you're cool with that I am too, maybe rope in Danny on the decision.